external/model-preparation-algorithm/configs/classification/efficientnet_b0_cls_incr/model_multilabel.py

```diff
@@ -1,5 +1,5 @@
 _base_ = [
-    "../../../submodule/models/classification/ote_efficientnet_b0_multilabel.yaml",
+    "../../../submodule/models/classification/ote_efficientnet_b0_multilabel_al.yaml",
```

Comment:
what is "al" means? (It is not a name that can be understood at a glance.)

Reply:
it stands for "angular loss". I can change it to AAM. AAM - angular asymmetric margin loss, it is more consistent with our paper.

Reply:
But  we need again merge changes in MPA. I propose to leave it as is. al - angular loss